### PR TITLE
Adding a customer games page

### PR DESCRIPTION
**Pull request by ForestJay**
Thus far only Zachary and Flying Mollusk have responded.  For now we want everything on a single page, but that may change in the future.

### CODE REVIEW

**Comment by caboteria** [comment]
Why just games?  How about "Powered by Affdex" so we could include other non-gaming use cases?

**Comment by ForestJay** [comment]
@caboteria Well, I think we should really have one page per game/app, but that's a battle I'm not willing to fight at the moment.  Since I'm creating it, I'm focusing on games (since I'm only in contact with game developers).  I'm open to changing it though if people prefer to add non-games to the same page AND are going to add stuff.

**Comment by mostafazh** [comment]
I think it's better to list both apps and games there too.

File: pages/see_also/customergames.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: Customer Games
```
Comment (ahamino) [inline pages/see_also/customergames.md]: Powered By Affdex

File: pages/see_also/customergames.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+permalink: /customergames/
+metadata: false
+---
+
+<!---
+Eventually we hope to have one page per game.  Please limit to one image OR video and keep the description to under 1000 characters.
+-->
+
+# <a href="http://nevermindgame.com/">Nevermind</a>
```
Comment (ahamino) [inline pages/see_also/customergames.md]: Can we have a page per app .. then used %include% to include them all in the master apps.md.

**Comment by ahamino** [comment]
I prefer one page for all apps .. candidates for adding:

# Tega
http://robotic.media.mit.edu/portfolio/tega/

# Flinch
iOS: https://itunes.apple.com/us/app/flinch/id862550306?mt=8
Android: https://play.google.com/store/apps/details?id=com.spitfire.Flinch

# TrueCar LED
https://www.youtube.com/watch?v=IaeE9dHRleg
https://vimeo.com/137718045

# Hershey Sampler
http://wildbluetech.com/smile-sampler/

# Bentley Inspirator
http://www.popsci.com/bentleys-new-facial-recognition-app-will-recommend-you-car
iOS:  https://itunes.apple.com/app/id1026942999

# Media Rebel
http://www.mediarebel.com/

 # iMotions
https://imotions.com/video/affectiva-affdex-integration/

# AffdexMe
iOS: https://itunes.apple.com/eg/app/affdexme/id971529011?mt=8
Android: https://play.google.com/store/apps/details?id=com.affectiva.affdexme
Windows: http://affdex-sdk-dist.s3-website-us-east-1.amazonaws.com/windows/download_affdexme.html

**Comment by ForestJay** [comment]
@ahamino Can you please have those developers fill out the poll:
https://docs.google.com/forms/d/1ApslMiy_By702wrt-rf17VfDraSFsPIBC49kCr-FwsI/viewform

**Comment by ForestJay** [comment]
I strongly disagree with having our sample apps together with customer apps.  Sample apps are for educational purpose and are not at all in the same class as (for example) Nevermind.  Take a look at how Unity does this.  Unity has a "Made with Unity" portal for customers:

- https://madewith.unity.com/games/nevermind
- https://madewith.unity.com/games/assassins-creed-identity

Unity has sample apps in the Unity store:
- https://www.assetstore.unity3d.com/en/#!/content/40756
- https://www.assetstore.unity3d.com/en/#!/content/46209

I think Unity has a great model for how to do this and we should emulate it.

File: pages/see_also/customergames.md (hunk 2 above)
Comment (ForestJay) [inline pages/see_also/customergames.md]: @ahamino Does include work?  I tried various versions of include and none seemed to work (with Jekyll):  

http://stackoverflow.com/questions/4779582/markdown-and-including-multiple-files

Do you have an example syntax?

**Pull request merged**